doc/qbk/4.0.containers.qbk

```diff
@@ -0,0 +1,41 @@
+[/
+    Copyright (c) 2022 Alan de Freitas (alandefreitas@gmail.com)
+
+    Distributed under the Boost Software License, Version 1.0. (See accompanying
+    file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+
+    Official repository: https://github.com/CPPAlliance/url
+]
+
+[section:containers Containers]
+
+The library provides containers for representing URLs according to
+their generic syntax for well known schemes. The three main
+URL containers are:
+
+* __url_view__: A non-owning reference to a valid URL.
+* __url__: A modifiable container for a URL.
+* __static_url__: A URL with fixed-capacity storage.
```

Comment:
Consider reordering this from shortest -> longest. Having "url" sandwiched in the middle doesn't look very good. We should also add authority view:
```
* __url__: A modifiable URL.
* __url_view__: A non-modifiable reference to a URL.
* __static_url__: A modifiable URL with fixed capacity, inline storage.
* __authority_view__: Technically not a URL but still often useful.
```

And then we might say:
> The URL containers maintain a useful invariant: they always contain a valid URL.